libs/utils/utils.js

```diff
@@ -810,6 +812,10 @@ export async function loadIms() {
         clearTimeout(timeout);
       },
       onError: reject,
+      api_parameters: apiParams,
+      onAccessTokenHasExpired: accessTokenExpired,
+      onReauthAccessToken: reauthAccessToken,
+      onAccessToken: accessToken,
```

Comment:
You mentioned these are needed for DC, so I'm not convinced everyone needs to have access to them through utils. Have you considered a sort of additional IMS config that a consumer could set? As in send in an object with extra fields, iterate trough them and attach to the universal IMS config.

Reply:
I agree with @overmyheadandbody, sending an object and passing everything on - also makes it easier in case any consumer wants some different configs, or adds more configs in the future.

Reply:
Updated to use an `adobeid` object